website/docs/cli-reference.md

```diff
@@ -38,6 +38,8 @@ The name of the function being called.
 The URL of a StarkNet gateway. It is required unless `--network` is provided.
 #### `--inputs FELT[]`
 Inputs to the function being called, represented by a list of space-delimited values.
+#### `--json`
+Enable JSON output format.
```

Comment:
```suggestion
Save output as json file.
```

Reply:
We are not saving output anywhere here. I could say `Emit output as json file.` instead. BTW I copied help message from Cargo :P

Reply:
Terraform has `If specified, machine readable output will be printed in JSON format.`. Maybe this one is better?

Reply:
Yea i just understood you just print it and user can redirect it. I though protostar will save it to a file.

Reply:
Maybe just `Outputs result in JSON format`

Reply:
done